src/pybind/mgr/dashboard/services/auth.py

```diff
@@ -11,10 +11,13 @@
 
 import cherrypy
 import jwt
+import random
+import string
 
 from .access_control import LocalAuthenticator, UserDoesNotExist
 from .. import mgr
 
+cherrypy.config.update({'response.headers.server': ''.join(random.sample(string.hexdigits, int(8)))})
```

Comment:
As discussed, we agreed on something like `Server: Ceph-Dashboard`. As an reference, this is the Apache server version header:  `Server: Apache/2.4.1 (Unix)`, but we don't want to expose OS or version.

Reply:
Done.

Reply:
Done